src/web/components/global-nav/container/sub-nav/sub-nav.js

```diff
@@ -19,10 +21,18 @@ class SubNav extends Core {
 
     setModuleIndicatorName(name){
         this._findDOMEl('.hig__global-nav__container__sub-nav__left', this.el).textContent = name;
+        this._findDOMEl('.hig__global-nav__container__sub-nav__left.hig__global-nav__container__sub-nav__spacer', this.el).textContent = name;
```

Comment:
@recyclerobot Please notice this crazy stuff I'm doing to get the layout to work nicely. Happy to go over it with you if that would be helpful.